src/DistributedFFT.chpl

```diff
@@ -189,16 +113,20 @@ prototype module DistributedFFT {
       const (yDst, xDst, _) = DstDom.localSubdomain().dims();
       const myLineSize = zSrc.size*numBytes(T);
 
-      // Setup FFTW plans
+      // Set up FFTW plans
       var yPlan = setupBatchPlanColumns(T, ftType, {ySrc, zSrc}, parDim=2, signOrKind, FFTW_MEASURE);
       var xPlan = setupBatchPlanColumns(T, ftType, {xDst, zSrc}, parDim=2, signOrKind, FFTW_MEASURE);
       var zPlan = setup1DPlan(T, ftType, zSrc.size, 1, signOrKind, FFTW_MEASURE);
 
       // Use temp work array to avoid overwriting the Src array
       var myplane : [{0..0, ySrc, zSrc}] T;
 
-      forall iy in ySrc {
-        copy(myplane[0, iy, zSrc.first], Src[xSrc.first, iy, zSrc.first], myLineSize);
+      if useElegant {
```

Comment:
I wonder if we should change the variable name from `useElegant` to `usePrimitiveComms` or something like that.

Reply:
We could also make it a compile-time constant. I don't believe anyone other than you (or the Chapel team) will ever use the slow path!

Reply:
Yeah, I'd be on board with something like `config param usePrimitiveComms = true`